### PR TITLE
Update channels.yaml

### PR DESCRIPTION
**Pull request by pkchavva**
Adding a Slack channel [#finops](finops.org/) to enable folks in the Kubernetes community to engage and participate with the FinOps Community.
Right now, FinOps Slack is restricted to members who are certified. This channel is to allow the FInOps SIGs focused on Kubernetes(and containers) to have a broader conversation with the Kubernetes Community.

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: communication/slack-config/channels.yaml [hunk 1]
```diff
@@ -102,6 +102,7 @@ channels:
   - name: external-secrets
   - name: falco
   - name: fiaas
+   -name: finops
```
Comment (markyjackson-taulia) [inline communication/slack-config/channels.yaml:105]: The spacing is off with the -

**Review by markyjackson-taulia — CHANGES_REQUESTED** [review]

**Comment by markyjackson-taulia** [comment]
@castrojo wanted to check back on this

**Comment by markyjackson-taulia** [comment]
@castrojo circling back on this

**Comment by castrojo** [comment]
/approve

Sorry it took so long!

**Comment by jdumars** [comment]
@castrojo bad news - the checks are not passing

**Comment by markyjackson-taulia** [comment]
It now has just dawned on me that I was tagging @castrojo as the author but it really is @pkchavva 😮‍💨 

@pkchavva on line 110, you `-` is off, you need to line it up with the other `-` above, that will fix this.

File: communication/slack-config/channels.yaml (hunk 1 above)
Comment (pkchavva) [reply communication/slack-config/channels.yaml]: Done!

**Comment by pkchavva** [comment]
> It now has just dawned on me that I was tagging @castrojo as the author but it really is @pkchavva 😮‍💨
> 
> @pkchavva on line 105, your `-` is off, you need to line it up with the other `-` above, that will fix this.

Let me know if this worked.

My first contribution so wasn't sure how to submit a patch.

**Review by markyjackson-taulia — APPROVED** [review]
/lgtm

**Pull request merged**